plugins/alerts/frontend/public/javascripts/countly.views.js

```diff
@@ -627,7 +635,7 @@ window.AlertsView = countlyView.extend({
             var selectedSingleAPP = $("#single-app-dropdown").clySelectGetSelection();
             settings.selectedApps = selectedSingleAPP ? [selectedSingleAPP] : null;
 
-            settings.compareDescribe = settings.alertDataSubType + (settings.alertDataSubType2 ? ' (' + settings.alertDataSubType2 + ')' : '') +
+            settings.compareDescribe = settings.alertDataSubType + (settings.alertDataSubType2 ? ' (' + document.querySelector('div[data-value="' + settings.alertDataSubType2 + '"]').textContent + ')' : '') +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://app.codacy.com/app/ar2rsawseen/countly-server/pullRequest?prid=4574400)